parser/proto.y

```diff
@@ -98,29 +92,26 @@ import (
 %type <sl>           listElements messageLiterals
 %type <msgLitFlds>   messageLiteralFieldEntry messageLiteralFields messageTextFormat
 %type <msgLitFld>    messageLiteralField
-%type <fld>          messageFieldDecl oneofFieldDecl extensionFieldDecl
+%type <msgFld>       messageFieldDecl
+%type <fld>          oneofFieldDecl extensionFieldDecl
 %type <oo>           oneofDecl
 %type <grp>          groupDecl oneofGroupDecl
+%type <msgGrp>			 messageGroupDecl
```

Comment:
nit: indentation (probably tab staps in here instead of just spaces)
Here and a few places below.